autofill/autofill-impl/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.duckduckgo.autofill.impl">
 
+    <uses-permission android:name="android.permission.CREDENTIAL_MANAGER_SET_ORIGIN" />
```

Comment:
this is required for browsers as we're a special kind of app that is looking to interact with passkeys based on URLs (origins) rather than app package IDs. Google has us listed as an app with privileges to do this (most general apps cannot).